packages/logging/service.go

```diff
@@ -89,6 +128,71 @@ func (s *loggingService) GetForDogu(request *pb.DoguLogMessageRequest, server pb
 	return writeLogLinesToStream(s.logProvider, doguName, linesCount, server)
 }
 
+func (s *loggingService) SetLogLevel(ctx context.Context, req *pb.LogLevelRequest) (*emptypb.Empty, error) {
```

Comment:
As mentioned in cloudogu/ces-control-api#6 the function does way more than just setting a value. The method name should hint at the possible downtime. The exported method has also no doc comment so I suggest something along the lines of the following (analguous to my suggestion in cloudogu/ces-control-api#6):

```suggestion
// ApplyLogLevelWithRestart sets the log level for a specific dogu and restarts the dogu if the log level was changed.
func (s *loggingService) ApplyLogLevelWithRestart(ctx context.Context, req *pb.LogLevelRequest) (*emptypb.Empty, error) {
```